### ISSUE TITLE
Dynamic card width

### ISSUE THREAD

**Issue by agronick** [Jeffreyrn/vue-simple-flowchart#2]
It would be nice if the arrows still aligned when the width of the card was increased via css.

Also it would be nice to be able to control the contents of the card with a slot.

**Comment by Zomis** [comment]
I totally agree about the slot thingy, I was just disappointed that it only supports text as content.

I would say that this issue can be renamed to simply "Dynamic card" or "Dynamic node".

**Comment by agronick** [comment]
I thinks thats changed. Slots support anything now.

**Comment by Zomis** [comment]
Vue slots in general supports anything, yes. But this project isn't using slots. Neither in https://github.com/Jeffreyrn/vue-simple-flowchart/blob/master/src/components/FlowchartNode.vue or in https://github.com/Jeffreyrn/vue-simple-flowchart/blob/master/src/components/SimpleFlowchart.vue, which is a big reason for me to not use this.

**Comment by agronick** [comment]
I misunderstood. I thought you were the maintainer. Yeah slots would open this up to a lot more use cases.

**Comment by ovictoraurelio** [comment]
Are you using another project to work with flowchart? Wich one?
